content/pages/airport-cases.md

```diff
@@ -0,0 +1,41 @@
+Title: The Airport Cases
+Date: 2017-01-31 **:**
+Author: anseljh
+Slug: airport-cases
+Status: draft
+
+On January 27, 2017, President Trump signed an [Executive Order titled "Protecting the Nation from Foreign Terrorist
+Entry Into the United States"](https://assets.documentcloud.org/documents/3431047/Extreme-Vetting-EO.pdf). The next day, civil rights and immigration lawyers began filing legal challenges throughout the country. As the dockets from these cases become available in Free Law Project's [RECAP Archive](https://www.courtlistener.com/recap/), we are linking them on this page.
+
+## Dockets by Circuit
+
+- First Circuit
+  - District of Massachusetts
+    - Tootkaboni v. Trump (case no. 17-cv-10154)
+- Second Circuit
+  - Eastern District of New York
+    - [Darweesh v. Trump (case no. 1:17-cv-480)](https://www.courtlistener.com/docket/4576731/darweesh-v-trump/)
+- Fourth Circuit
+  - Eastern District of Virginia
+    - [Aziz v. Trump](https://www.courtlistener.com/docket/4576795/aziz-v-trump/) (case no. 1:17-cv-116)
+- Ninth Circuit
+  - Central District of California
+    - [Vayeghan v. Kelly (case no. 2:17-cv-00702)](https://www.courtlistener.com/docket/4576959/ali-khoshbakhti-vayeghan-v-john-f-kelly/)
+  - Western District of Washington
+    - [Doe v. Trump](https://www.courtlistener.com/docket/4577105/doe-1-v-trump/) (case no. 2:17-cv-126) &mdash; Voluntarily dismissed
+
+## Updating Dockets
```

Comment:
To update a docket, [install the RECAP extension][recap] and download the latest dockets and PDFs from PACER. RECAP will send us an updated version of the information, and It will automatically be updated here within a few hours (processing can take some time).

[recap]: https://free.law/recap/